resources/metadata/cloudFoundryCreateService.yaml

```diff
@@ -0,0 +1,157 @@
+metadata:
```

Comment:
Please make the capitalisation and parameter descriptions more consistent. For example:

- CF -> Cloud Foundry
- Cloud Foundry service manifest -> Cloud Foundry Service Manifest